src/graphqlv2/schemas/inputs/searchString.graphql

```diff
@@ -10,4 +10,5 @@ input SearchString {
     above: Boolean
     below: Boolean
     notEquals: SearchStringValue
+    contains: String
```

Comment:
support for "exact" modifier also needs to be added
If you think we should split this then please create separate ticket for this

Reply:
Will open separate ticket